test/blackbox-tests/test-cases/directory-targets/github6168.t/fakenpm.ml

```diff
@@ -0,0 +1,5 @@
+let () =
+  Sys.mkdir "fakenode_modules" 0o777;
+  Sys.mkdir "fakenode_modules/foo" 0o777;
+  Unix.symlink "file" "./fakenode_modules/foo/bar";
+  ()
```

Comment:
why do you have this terminating `()`?

Reply:
It's just a style to help indicate the end of a bunch of semicolons. I've removed it.